server/channels/api4/integration_action_test.go

```diff
@@ -6,6 +6,8 @@ package api4
 import (
 	"context"
 	"encoding/json"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+	"github.com/mattermost/mattermost/server/v8/channels/testlib"
```

Comment:
golangci doesn't like the order of imports

```
channels/api4/integration_action_test.go:9: File is not `goimports`-ed (goimports)
	"github.com/mattermost/mattermost/server/public/shared/mlog"
	"github.com/mattermost/mattermost/server/v8/channels/testlib"
make: *** [Makefile:290: golangci-lint] Error 1
```

These two imports should be next to the `"github.com/mattermost/mattermost/server/public/model"` import.

If you use VSCode with the Go extension, it automatically formats the import rules.

Reply:
I'm using GoLand :cry: